toast.css

```diff
@@ -0,0 +1,70 @@
+/* The snackbar - position it at the bottom and in the middle of the screen */
+
+  .toast{
+    width: fit-content;
+    padding: 1rem 3rem 1rem 1rem ;
+    /* top:1rem; */
+    /* right: 1rem; */
+    background-color: white;
+    color: var(--secondary-text-color);
+    border-radius: 0.3rem;
+    /* display: none; */
+    z-index: var(--z-index-5);
+	/* visibility: hidden; Hidden by default. Visible on click */
+  
+  
+}
+
+.toast-cancel-btn{
+	margin-left: 3rem;
+}
+.target-toast{
+	display: none;
+}
+.toast-success{
+    background-color: #84f1c9;
+    color: #1b7052;
+}
+.toast-error{
+    color: #dc3545;
+    background-color: #fcc5c5;
+}
+.toast-inform{
+    background-color: #7FD2E6;
+    color: #29438b;
+}
+  
+  /* Show the snackbar when clicking on a button (class added with JavaScript) */
+  .show {
+	visibility: visible; /* Show the snackbar */
+	    position: fixed;
+
+	left: 50%; /* Center the snackbar */
+  bottom: 30px; /* 30px from the bottom */
+	display: block;
+	/* Add animation: Take 0.5 seconds to fade in and out the snackbar.
```

Comment:
remove commented code

Reply:
I have left the comments for future references and easy understanding.